scripts/run-app.sh

```diff
@@ -1,55 +1,56 @@
 #!/usr/bin/env sh
 
 log() {
-  local message=$1
-  local iso_8601_timestamp=$(date +%Y-%m-%dT%H:%M:%S%z)
-  echo '{"@version": 1, "@timestamp": "'"$iso_8601_timestamp"'", "level": "INFO", "message": "'"$1"'"}'
+  message=$1
```

Comment:
shellscript newbie, but are variables automatically scoped to their function?

Reply:
nope, globals all the way. I think there's no way to avoid this in `sh` if I can't use `local`, except running everything as different processes

Reply:
You can use subshells, but ugh. `local` is fine as long as you don't care about Solaris and other OS-es where /bin/sh isn't POSIX sh.

Reply:
Also, quoting.

Reply:
/bin/sh points to busybox here, which supports local. Should I just use that and ignore Shellcheck?

Reply:
Yeah, and use https://github.com/koalaman/shellcheck/wiki/Directive if possible to disable just the `local` warning?

Reply:
Or follow the advice on https://github.com/koalaman/shellcheck/wiki/SC2155 ? I assume that's what it's warning about, not some other error?

Reply:
It's warning about https://github.com/koalaman/shellcheck/wiki/SC2039:

For example, adding `local` in front of `message="$1"` on line 4:

```
In scripts/run-app.sh line 4:
  local message="$1"
  ^-- SC2039: In POSIX sh, 'local' is undefined.
```

Reply:
That'd I'd just silence, yeah.